discovery-provider/src/monitors/redis.py

```diff
@@ -0,0 +1,22 @@
+import psutil
+
+
+def get_redis_num_keys(**kwargs):
+    redis = kwargs['redis']
+    num_keys = redis.dbsize()
+    return num_keys
+
+
+def get_redis_used_memory(**kwargs):
+    redis = kwargs['redis']
+    info = redis.info()
+    return info['used_memory']
+
+
+def get_redis_total_memory(**kwargs):
+    # The version of redis used does not support total_system_memory
+    # redis = kwargs['redis']
```

Comment:
is this comment for a future version of redis...?

Reply:
yep! added more comments around this